cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -293,7 +326,7 @@ def test_conversion_host_tags(appliance, v2v_providers):
             display_name='V2V - Transformation Method')
             .collections.tags.instantiate(display_name='VDDK'))
 
-    host = v2v_providers[1].hosts.all()[0]
+    host = v2v_providers.rhv_provider.hosts.all()[0]
```

Comment:
what is the 0th index here? Same comment as about, lets put this clarification

Reply:
added comment